keps/2724-topology-aware-scheduling/README.md

```diff
@@ -539,6 +698,19 @@ type PodSetTopologyRequest struct {
   // SubGroupIndexLabel indicates the count of replicated Jobs (groups) within a PodSet.
   // For example, in the context of JobSet this value is read from jobset.sigs.k8s.io/replicatedjob-replicas.
   SubGroupCount *int32
+
+  // PodSetSliceRequiredTopology indicates the topology level required by the PodSet slice, as
+  // indicated by the `kueue.x-k8s.io/podset-slice-required-topology` annotation.
+  //
+  // +optional
+  PodSetSliceRequiredTopology *string `json:"podSetSliceRequiredTopology,omitempty"`
+
+  // PodSetSliceSize indicates the size of a subgroup of pods in a PodSet for which
+  // Kueue finds a requested topology domain on a level defined
+  // in `kueue.x-k8s.io/podset-slice-required-topology` annotation.
+  //
+  // +optional
+  PodSetSliceSize *int32 `json:"podSetSliceSize,omitempty"`
```

Comment:
```yaml
topologyRequest:
  podSetSlice:
    requiredTopology: xxx
    size: yyy
    preferred: zzz // This is not included in this proposal. for future expanding.
```

Do we need to consider future API expansion?
@mimowo Do you have any plans to expand the PodSet Slice capability in TAS?

Reply:
> @mimowo Do you have any plans to expand the PodSet Slice capability in TAS?

No,  the current API proposal is already covering all use-cases we currently have regarding PodSet slices.

There might be future extensions which are hard to predict as typically with future :).

Personally, I think flat structure as in the current KEP is ok for simplicity.

Reply:
Alright. In that case, let's keep the current form.